R/derive_param_computed.R

```diff
@@ -149,12 +149,27 @@
 #'
 #' @export
 #'
-#' @examples
+#' @examplesx
+#'
+#' @caption Example 1a - Adding a parameter computed from a formula
+#'
+#' @info Derive mean arterial pressure (MAP) from systolic (SYSBP)
+#'     and diastolic blood pressure (DIABP)
+#'
+#' - Here, for each `USUBJID` and `VISIT` group, an observation is added
+#'   to the output dataset when the filtered input dataset (`dataset`)
+#'   contains exactly one observation for each parameter code specified
+#'   for `parameters` and all contributing values (e.g., `AVAL.SYSBP`
+#'   and `AVAL.DIABP`) are not `NA`.
+#' - This results in three additional rows: `USUBJID` `01-701-1015` at
```

Comment:
You can avoid explicitly mentioning the new rows as they are visible right below the text.